Source/WebKit/Shared/WebCoreArgumentCoders.serialization.in

```diff
@@ -2546,6 +2546,24 @@ header: <WebCore/KeyboardScroll.h>
     WebCore::TransformationMatrix matrix();
 }
 
+#if PLATFORM(COCOA)
+struct AudioStreamBasicDescription {
+    double    mSampleRate;
+    uint32_t  mFormatID;
```

Comment:
Looks like we'll have to use UInt32 to keep watchOS building.  Classic int vs long